config.yaml

```diff
@@ -80,6 +80,8 @@ service:
     - standard
     - medium
     - large
+  catalog:
+    - https://github.com/platformsh/template-drupal7-vanilla/blob/master/.platform.template.yaml  
```

Comment:
This file has been removed.  (Or, well, it's supposed to have been removed. We may have some cleanup to do yet.)  The template.yaml files are no longer included in the Git repository since they don't have any real value there.  They're in the template-builder repository for the first-party templates.

Also, a catalog is a collection of such template definitions, no?  Not a single entry?

Reply:
I believe we should not need a default catalog file here. This should be using the `/setup/catalog` API and the API is responsible (based on the user's OEM settings, vendor, and organization) for providing the correct default catalog file.

Reply:
Looks like this is provided solely as a fallback in the case that the CLI has no runtime in which case it should be `http://accounts.platform.sh/sites/default/files/catalog/catalog-psh.yaml`.

Reply:
That was just a place holder for testing, I've updated it to point to the correct file.